### DISCUSSION TITLE
Change icon

### DISCUSSION THREAD

**Discussion by RasputinBerry** [arendst/Tasmota#17819] [Q&A] [answered]
How to change the default icon shown in the browser? 
It looks (mostly) boring and should have something to do with the respective application...
![grafik](https://user-images.githubusercontent.com/56359831/215321221-9c984f07-f524-4dd9-8e52-36e0789adc97.png)

**Comment by sfromis** [comment]
Tasmota does not provide a feature to replace the default favicon, unless you modify the source. It is a lot less boring than when there was no favicon.

**Reply by barbudor** [reply]
It has to do with the application as it is Tasmota logo
😱

**Reply by sfromis** [reply]
While the favicon works well when show in the browser, it can be said to be rather "sub-optimal" when creating a desktop shortcut (or whatever it may be called), where the small favicon is scaled up to fill the much larger space intended for application icons.

**Reply by barbudor** [reply]
If I came with a multiple resolution icon you know what would have happen to my PR. 64 bytes was the max that Theo was ready to accept :rofl:
On a desktop shortcut you can still point to a better resolution local icon file

**Reply by sfromis** [reply]
Yes, I know :grinning: 

I was mainly reacting to the original image posted in this thread, which does not look like being from a browser, but instead from a desktop bookmark, and how it visually does not have a good look due to being scaled up by a lot. And yes, If I had a habit of such shortcuts.....
![image](https://user-images.githubusercontent.com/47082390/215330848-673be82e-55bc-4082-8ca2-d052ab85843c.png)

**Comment by RasputinBerry** [comment, marked as answer]
OK ... not to be changed afterwards.
Actually doesn't really matter. 
It would have been nice as a distinguishing feature (e.g. with the bookmarks).
